tasks/install.yml

```diff
@@ -3,20 +3,20 @@
 
 - include: pip_index.yml
 
-- name: Install deps to avoid pip doing compilation or enable it
-  apt: name={{ item }} state=present
-  with_items: "{{ dependencies_deb }}"
-  when: ansible_os_family == 'Debian'
-
-- name: Install rpms to avoid pip doing compilation or enable it
-  yum: name={{ item }} state=present
-  with_items: "{{ dependencies_redhat }}"
-  when: ansible_os_family == 'RedHat'
+- name: Set OS distribution dependent variables
+  include_vars: "{{ lookup('first_found', params) }}"
+  vars:
+    params:
+      files:
+        - "deps_{{ ansible_os_family }}_{{ ansible_distribution_major_version }}.yml"
```

Comment:
Much cleaner, thanks. I haven't tested it on the various distros, but hopefully this is enough. They probably need attention anyway.